pkgs/development/python-modules/pyregion/default.nix

```diff
@@ -33,9 +33,8 @@ buildPythonPackage rec {
   # Upstream patch needed for the test to pass
   patches = [
     (fetchpatch {
-      name = "conftest-astropy-3-fix.patch";
-      url = "https://github.com/astropy/pyregion/pull/136.patch";
-      sha256 = "13yxjxiqnhjy9gh24hvv6pnwx7qic2mcx3ccr1igjrc3f881d59m";
+      url = "https://github.com/astropy/pyregion/pull/157.patch";
```

Comment:
Since the PR is not merged yet, we must fetch each commit individually https://github.com/astropy/pyregion/pull/157/commits

Reply:
Fixed with acc3247.